libs/designer-ui/src/lib/settings/settingsection/settingTokenField.tsx

```diff
@@ -42,6 +42,7 @@ export interface SettingTokenFieldProps extends SettingProps {
   expressionGroup?: TokenGroup[];
   isTrigger?: boolean;
   isCallback?: boolean;
+  editorValueType?: string;
```

Comment:
Nobody is using this and we should not be passing this as props to settingtoken field, because editors internally should get their sub types from editorOptions